src/devices/machine/ibmps2.cpp

```diff
@@ -0,0 +1,1113 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol, Miodrag Milanovic, Carl
+/***************************************************************************
+
+    IBM PS2 Compatibles
+
+***************************************************************************/
+
+#include "emu.h"
```

Comment:
You should put the module header immediately after emu.h, before the headers for all the other things it uses.